tezster.js

```diff
@@ -84,8 +84,13 @@ program
     const tezsterManager = require('./tezster-manager');    
     await tezsterManager.loadTezsterConfig();
     const config = tezsterManager.config;
+    if(Object.keys(config.accounts).length > 1){
```

Comment:
Why do you need to check for config.accounts.length >1 ?

Reply:
because if, there are no contracts available then, previously it shows nothing as output, the end user will never what was happened.
when I'm doing this, then if contracts are available then it'll show the list. or in the case of the empty list, it shows response like "no contracts is avaiablel"

Reply:
@jvtezsure

Reply:
Sorry, "config.accounts.length >1" this should be greater than Zero, I made changes according to it.